scripts/common/prompts.sh

```diff
@@ -2,36 +2,18 @@
 # Gather any additional information required from the user that could not be discovered and was not
 # passed with a flag
 
-function prompts_can_prompt() {
-    # Need the TTY to accept input and stdout to display
-    # Prompts when running the script through the terminal but not as a subshell
-    if [ -t 1 ] && [ -c /dev/tty ]; then
-        return 0
-    fi
-    return 1
-}
-
 function prompt() {
-    if ! prompts_can_prompt ; then
-        bail "Cannot prompt, shell is not interactive"
-    fi
-
     set +e
-    read PROMPT_RESULT < /dev/tty
+    read -p "(y/n)?" PROMPT_RESULT
     set -e
 }
 
 function confirmY() {
-    printf "(Y/n) "
     if [ "$ASSUME_YES" = "1" ]; then
+        printf "(Y/n) "
         echo "Y"
         return 0
     fi
-    if ! prompts_can_prompt ; then
-        echo "Y"
-        logWarn "Automatically accepting prompt, shell is not interactive"
```

Comment:
What happens now for a non-interactive shell? Say i were to invoke the kurl script from cloud init. I would expect confirmY to continue and confirmN to exit unless ASSUME_YES.